drivers/SmartThings/matter-button/src/init.lua

```diff
@@ -13,17 +13,22 @@ local STATIC_PROFILE_SUPPORTED = {2, 4, 8}
 
 local COMPONENT_TO_ENDPOINT_MAP = "__component_to_endpoint_map"
 local DEFERRED_CONFIGURE = "__DEFERRED_CONFIGURE"
+
+-- Some switches will send a MultiPressComplete event as part a long press sequence. Normally the driver will create a
+-- button capability event on receipt of MultiPressComplete, but in this case that would result in an extra event because
+-- the "held" capability event is generated when the LongPress event is received. The IGNORE_NEXT_MPC flag is used
+-- to tell the driver to ignore MultiPressComplete if it is received after a long press to avoid this extra event.
 local IGNORE_NEXT_MPC = "__ignore_next_mpc"
```

Comment:
So this flag was here but never used? Hmm...

Reply:
It was being used, but it was ignore the next MultiPressComplete whether it came following a long press (InitialPress, LongPress, LongRelease, MultiPressComplete) or it was part of a separate press event following a long press (long press: InitialPress, LongPress, LongRelease, single press: InitialPress, ShortRelease, MultiPressComplete). The spec does seem to require MultiPressComplete to be sent as part of a long press event sequence, but not all devices are doing that and there has been some discussion that the spec needs to be more clear in this area.